build-support/travis/travis.yml.mustache

```diff
@@ -234,11 +234,12 @@ base_linux_build_engine: &base_linux_build_engine
     # Note that:
     # * We mount ${HOME} to cache the ${HOME}/.cache/pants/rust-toolchain.
     # * We also build fs_util, to take advantage of the rust code built during bootstrapping.
+    # "${HOME}/.bashrc" is converted into /home/travis for some reason, so we source the full path (using "." instead of "source" for portability).
     - docker run --rm -t
       -v "${HOME}:/travis/home"
       -v "${TRAVIS_BUILD_DIR}:/travis/workdir"
       travis_ci:latest
-      sh -c "./build-support/bin/ci.sh ${BOOTSTRAP_ARGS} && ./build-support/bin/release.sh -f"
+      sh -c ". /travis/home/.bashrc && ./build-support/bin/ci.sh ${BOOTSTRAP_ARGS} && ./build-support/bin/release.sh -f"
```

Comment:
We now use Docker in the build wheels shard too. I think we will need to update that .travis.yml entry as well with this snippet.

It's difficult to test because Build Wheels fails for everyone atm..

Reply:
e5b2bd8 should have done this!